apis/project_api/project_api/utils.py

```diff
@@ -173,7 +173,7 @@ def load_uhs_data(results_dir: Path, rps: List[int]):
     # Need to fix the `uhs_nz11750` before we generate new project data
     nzs1170p5_results = [
         sc.nz_code.nzs1170p5.NZS1170p5Result.load(cur_dir, ensemble=ensemble)
-        for cur_dir in (results_dir / "uhs_nz11750").glob("uhs_*")
+        for cur_dir in (results_dir / "uhs_nzs1170p5").glob("uhs_*")
```

Comment:
Yeah, this was the line I shared either in one of the previous PR or slack that projects under Mantle path was also `uhs_nz11750` and you would get an error due to naming updated. (Thought I updated :thinking: )

And it seems someone already renamed the directory, thanks to whoever did this.

While you are updating this, would you also be able to remove the comment above? line number 173, thanks!